ofl/notonaskharabic/METADATA.pb

```diff
@@ -1,7 +1,7 @@
 name: "Noto Naskh Arabic"
 designer: "Google"
 license: "OFL"
-category: "SANS_SERIF"
+category: "SERIF"
```

Comment:
Intentional change?

Reply:
Yes, we decided this was better paired with a Serif latin.

Reply:
Ty!